### DISCUSSION TITLE
Welcome to langchain-rust Discussions!

### DISCUSSION THREAD

**Discussion by Abraxas-365** [Abraxas-365/langchain-rust#90] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by corysabol** [comment]
Hello, I just found this project and I'm considering integrating it into a project of mine. I was wondering if I could get some guidance on creating my own LLM that can be used with the chains. To give some more details, I have written some code in Rust using Candle to do local inference and I would like to port that code to work with this project. If it would be useful for others to be able to use this project to do purely local inference without relying on additional APIs I would be happy to contribute it back upstream as well. Assuming I get it working :)
